src/Tizen.Applications.Common/Tizen.Applications/Bundle.cs

```diff
@@ -62,11 +62,16 @@ public Bundle()
         /// <since_tizen> 3 </since_tizen>
         public Bundle(SafeBundleHandle handle)
         {
-            if (handle == null || handle.IsInvalid)
+            if (handle == null)
             {
                 throw new ArgumentNullException("handle");
             }
 
+            if (handle.IsInvalid)
+            {
+                throw new ArgumentNullException("handle is invalid");
```

Comment:
```suggestion
                throw new ArgumentNullException("handle", "handle is invalid");
```

https://docs.microsoft.com/ko-kr/dotnet/api/system.argumentnullexception.-ctor?view=netframework-4.8#System_ArgumentNullException__ctor_System_String_System_String_

Reply:
Done